networks/rpc/service.go

```diff
@@ -104,12 +104,19 @@ func (r *serviceRegistry) callback(method string) *callback {
 	r.mu.Lock()
 	defer r.mu.Unlock()
 
+	namespace, method := elem[0], elem[1]
+
 	// when NonEthCompatible is true, the return formatting for the eth namespace API provided for Ethereum compatibility is disabled.
 	// convert ethereum namespace to klay namespace.
-	if NonEthCompatible && elem[0] == "eth" {
-		return r.services["klay"].callbacks[elem[1]]
+	if NonEthCompatible && namespace == "eth" {
+		return r.services["klay"].callbacks[method]
+	}
+
+	if namespace == "kaia" {
+		return r.services["klay"].callbacks[method]
```

Comment:
for now, using an alias, but will It be changed to just using 'kaia' without an alias?

Reply:
Yes, allowing `klay_` is for ecosystem to prepare the merge, and we expect gradual transition to `kaia_`. We'll announce them if there're noticeable changes.